package.json

```diff
@@ -114,10 +115,12 @@
     "swiper": "^11.0.7",
     "tailwind-merge": "^2.2.1",
     "tailwindcss": "^3.4.3",
-    "typescript": "^5.3.3",
```

Comment:
I think we shouldn't remove typescript here because it will be used as dependency for Next js, IMO.